lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -123,125 +135,135 @@ table tr:nth-child(even) td {
+nav[hidden] {
   display: none;
 }
 
-#search-field {
-  width: 98%;
-  background: white;
-  border: none;
-  height: 1.5em;
-  -webkit-border-radius: 4px;
-  -moz-border-radius: 4px;
-  border-radius: 4px;
-  text-align: left;
-}
-#search-field:focus {
-  background: #f1edba;
-}
-#search-field:-moz-placeholder,
-#search-field::-webkit-input-placeholder {
-  font-weight: bold;
-  color: #666;
+nav footer {
+  padding: 1em;
+  border-top: 1px solid #ccc;
 }
 
-.missing-docs {
-  font-size: 120%;
-  background: white url(../images/wrench_orange.png) no-repeat 4px center;
-  color: #ccc;
-  line-height: 2em;
-  border: 1px solid #d00;
-  opacity: 1;
-  padding-left: 20px;
-  text-indent: 24px;
-  letter-spacing: 3px;
-  font-weight: bold;
-  -webkit-border-radius: 5px;
-  -moz-border-radius: 5px;
+nav .nav-section {
+  margin-top: 1em;
+  padding: 0 1em;
 }
 
-.target-section {
-  border: 2px solid #dcce90;
-  border-left-width: 8px;
-  padding: 0 1em;
-  background: #fff3c2;
+nav h2 {
+  margin: 0 0 0.5em;
+  padding: 0.5em 0;
+  font-size: 1.2em;
+  color: var(--text-color);
+  border-bottom: 1px solid var(--border-color);
 }
 
-/* @end */
+nav h3,
+#table-of-contents-navigation {
+  margin: 0;
+  padding: 0.5em 0;
+  font-size: 1em;
+  color: var(--text-color);
+}
 
-/* @group Index Page, Standalone file pages */
-.table-of-contents ul {
-  margin: 1em;
+nav ul,
+nav dl,
+nav p {
+  padding: 0;
   list-style: none;
+  margin: 0.5em 0;
 }
 
-.table-of-contents ul ul {
-  margin-top: 0.25em;
+nav ul li {
+  margin-bottom: 0.3em;
 }
 
-.table-of-contents ul :link,
-.table-of-contents ul :visited {
-  font-size: 16px;
+nav ul ul {
+  padding-left: 1em;
 }
 
-.table-of-contents li {
-  margin-bottom: 0.25em;
+nav ul ul ul {
+  padding-left: 1em;
 }
 
-.table-of-contents li .toc-toggle {
-  width: 16px;
-  height: 16px;
-  background: url(../images/add.png) no-repeat;
+nav ul ul ul ul {
+  padding-left: 1em;
 }
 
-.table-of-contents li .toc-toggle.open {
-  background: url(../images/delete.png) no-repeat;
+nav a {
+  color: var(--link-color);
+  text-decoration: none;
 }
 
-/* @end */
+nav a:hover {
+  text-decoration: underline;
+}
 
-/* @group Top-Level Structure */
+#navigation-toggle {
+  z-index: 1000;
+  font-size: 2em;
+  position: absolute;
+}
 
-nav {
-  font-family: Helvetica, sans-serif;
-  font-size: 14px;
-  border-right: 1px solid #ccc;
-  position: fixed;
-  top: 0;
-  bottom: 0;
-  overflow: auto;
-  z-index: 10;
+#navigation-toggle[aria-expanded="true"] {
+  top: 10px;
+  left: 250px;
+}
 
-  /* Layout */
-  width: var(--sidebar-width);
-  min-height: 100vh;
-  background: white;
+#navigation-toggle[aria-expanded="false"] {
+  top: 10px;
+  left: 20px;
+}
 
-  display: flex;
-  flex-direction: column;
+nav ul li details {
+  position: relative;
+  padding-right: 1.5em;  /* Add space for the marker on the right */
 }
 
-nav[hidden] {
-  display: none;
+nav ul li details > summary {
+  list-style: none;  /* Remove the default marker */
```

Comment:
Unfortunately, this still doesn't work in Safari 18.1.
`summary::-webkit-details-marker {display: none;}` is needed.